cluster/evaluator/Dockerfile-cc

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/$PROJECT/cc-base:latest
+FROM gcr.io/$PROJECT/cc-base:0.17
```

Comment:
this is fine, you're welcome to commit as-is; you could also have changed just this one to use 0.1.